axe.d.ts

```diff
@@ -146,6 +153,14 @@ declare namespace axe {
 		}[];
 		cleanup?(callback: Function): void;
 	}
+	interface VirtualNode {
+		children?: VirtualNode[];
```

Comment:
This means `children` is either `undefined` or an array of `VirtualNode`s. IMO it should _always_ be an array, even if the array is empty.

This way you don't have to:

```
if (vNode.children && vNode.children.length) {
 // ...
}
```